app/Controller/PagesController.php

```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
+
+/**
+ * TooManyMethods
+*/
+public function display1() {
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected 4 spaces, found 0
* Expected 2 blank lines before function; 1 found
* Opening brace should be on a new line